package.json

```diff
@@ -61,6 +61,7 @@
     "isomorphic-cookie": "^1.2.4",
     "jwt-decode": "^3.1.2",
     "lemmy-js-client": "0.17.2-rc.17",
+    "lodash": "^4.17.21",
```

Comment:
lodash seems unused, and I'd def like to avoid it if possible.

Reply:
`lodash` is only being used in `webpack.config.js`. I originally tried having it as a dev dependency, but this caused issues when I tested the prod Dockerfile. This shouldn't be included in the bundle.

Reply:
yeah i got this issue too when trying to run docker compose against lemmy-ui